examples_tests/61.BlitFilterTest/normalization/parameters.glsl

```diff
@@ -0,0 +1,12 @@
+#ifndef _NBL_GLSL_BLIT_NORMALIZATION_PARAMETERS_INCLUDED_
+#define _NBL_GLSL_BLIT_NORMALIZATION_PARAMETERS_INCLUDED_
+
+struct nbl_glsl_blit_normalization_parameters_t
+{
+	uvec3 outImageDim;
+	uint padding;
+	uint inPixelCount;
+	float oldReferenceAlpha;
+};
+
+#endif
```

Comment:
delete the whole file, lets unify the push constants into a single struct

Reply:
Resolved.